### ISSUE TITLE
Allow library to be called using python -m 

### ISSUE THREAD

**Issue by ice3** [staticjinja/staticjinja#152]
Hello, in some environments, the scripts are not in the path so the "classical" way to call a python package is to use `python -m ...`. 

With staticjinja, it doesn't work as expected: the cli arguments are not considered. In my case, it doesn't consider the `outpath` nor the `build` when doing `python -m staticjinja build --outpath=build`

The root cause seems to be that `python -m`will run the `__main__.py` file which doesn't call the `main` function from `cli.py` (the main function parses the parameters). 

I fixed this by replacing the code of `__main__` with : 

```python
#!/usr/bin/env python

import os

from .staticjinja import Site
from .cli import main

if __name__ == "__main__":
    main()
```

I don't know if this is file has to handle other cases that would be broken by the change. 

If this seems OK, I can submit a PR with the change.

**Comment by NickCrews** [comment]
Hi! Thanks for the issue. I know exactly what you mean.

I've thought before about changing this, and in the past I've been hesitant to do so because I worried that I would break someone's workflow. But, it's good to hear from someone that the current code _is not_ working for them. I'm going to use this to justify to myself that it's OK to make the change: It might break someone, but hopefully it fixes things for someone else ;)

So if you want, it would be great if you wanted to submit a PR. Just a nit, I think the snippet you have above has some unneeded imports, I think just 
```
#!/usr/bin/env python

from .cli import main

if __name__ == "__main__":
    main()
```
is what we want.

Thanks for the contribution!

**Comment by NickCrews** [comment]
Ok, just released this in 3.0.1, you should be able to update to that!
